src/components/common/Tabs/TargetProfileTab/styles.scss

```diff
@@ -13,6 +13,7 @@
     width: 125px;
     height: 125px;
     margin-bottom: 1rem;
```

Comment:
Instead of repeating this line for both margins don't you think it would be better to set this as `margin: 1rem 0` ?

Reply:
True!